cli/common/utils.go

```diff
@@ -274,3 +281,43 @@ func InsertInStringSlice(s []string, i int, elem string) []string {
 
 	return res
 }
+
+func GetInstancesFromArgs(args []string, ctx *context.Ctx) ([]string, error) {
+	foundInstances := make(map[string]struct{})
+	var instances []string
+
+	for _, instanceName := range args {
+		if instanceName == ctx.Project.Name {
+			return nil, fmt.Errorf(appNameSpecifiedError)
+		}
+
+		parts := strings.SplitN(instanceName, ".", 2)
+
+		if len(parts) > 1 {
+			return nil, fmt.Errorf(instanceIDSpecified)
+		}
+
+		if instanceName != "" {
+			if _, found := foundInstances[instanceName]; found {
+				return nil, fmt.Errorf("Duplicate instance name: %s", instanceName)
```

Comment:
"... found in config"?

Otherwise it could confuse me as I don't understand the root of problem.

Reply:
`Duplicate instance name specified`? It's parsing arguments specified by user.

Reply:
My bad. Agree